crypto/include/internal/asn1_int.h

```diff
@@ -63,6 +63,10 @@ struct evp_pkey_asn1_method_st {
     int (*set_pub_key) (EVP_PKEY *pk, const unsigned char *pub, size_t len);
     int (*get_priv_key) (const EVP_PKEY *pk, unsigned char *priv, size_t *len);
     int (*get_pub_key) (const EVP_PKEY *pk, unsigned char *pub, size_t *len);
+
+    /* Exports to providers */
```

Comment:
Perhaps we should add a TODO here (or somewhere else appropriate) to remind us to go and implement these for the various legacy keys that we have?

Reply:
Okie, added one here